test/test_datapipe.py

```diff
@@ -74,12 +65,7 @@
 dill = import_dill()
 HAS_DILL = TEST_DILL
 
-try:
-    import pandas  # type: ignore[import]  # noqa: F401 F403
-
-    HAS_PANDAS = True
-except ImportError:
-    HAS_PANDAS = False
+HAS_PANDAS: bool = importlib.util.find_spec("pandas") is not None
```

Comment:
Hah, didn't know `importlib.util.find_spec`, nice find which avoids the bogus import!

Reply:
Note that we have a utility function for that as well https://github.com/pytorch/pytorch/blob/main/torch/utils/_import_utils.py#L9